content/francais/python-turtle/activity-1.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Activité 1 - Dessiner un carré"
+date: 2024-09-17T13:24:17-07:00
+weight: 3
+draft: false
+---
+
+<img src="../media/bee_square.png" alt="Abeille imaginant un carré" width="25%" />
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/CRJf-LbXAx4" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
+
+Bravo ! Vous avez aidé Alex à dessiner une ligne ! Maintenant, passons à l’étape suivante et aidons Alex à dessiner un carré. Pour dessiner un carré, nous devons suivre ces étapes :
+
+1. Aller en avant de 50 pas
```

Comment:
It will make more sense if the verb "Aller" is conjugated in imperative (as an order). 

1. Va de 50 pas en avant.
2. Tourne de 90 degrés vers la droite.
3. Va de 50 pas en avant.
4. Tourne de 90 degrés vers la droite.
5. Va de 50 pas en avant.
6. Tourne de 90 degrés vers la droite.
7. Va de 50 pas en avant.

Reply:
updated